openpype/pipeline/colorspace.py

```diff
@@ -601,3 +601,63 @@ def _get_imageio_settings(project_settings, host_name):
     imageio_host = project_settings.get(host_name, {}).get("imageio", {})
 
     return imageio_global, imageio_host
+
+
+def get_display_view_colorspace_name(config_path, display, view):
+    """get view colorspace name for the given display and view.
```

Comment:
```suggestion
    """Return colorspace for the given display and view.
```

The wording _view colorspace_ somehow had me confused thinking that it wasn't a regular OCIO colorspace definition but something view-specific - which it isn't - so I'd recommend simplifying it like this.